src/main/java/de/craften/plugins/rpgplus/components/storage/PDSStorage.java

```diff
@@ -0,0 +1,80 @@
+package de.craften.plugins.rpgplus.components.storage;
+
+import java.util.Map;
+import java.util.UUID;
+
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+
+import de.craften.plugins.playerdatastore.api.PlayerDataStore;
+import de.craften.plugins.playerdatastore.api.PlayerDataStoreService;
+
+/**
+ * A storage that uses the PlayerDataStore.
+ */
+public class PDSStorage implements Storage{
+	
+	private PlayerDataStoreService getPDSService() {
+		return Bukkit.getServer().getServicesManager().getRegistration(PlayerDataStoreService.class).getProvider();
+	}
+	
+	private PlayerDataStore getStore(OfflinePlayer player) {
+		return getPDSService().getStore(player);
+	}
+	
+	private PlayerDataStore getStore(UUID uuid) {
+		return getPDSService().getStore(uuid);
+	}
+	
+	private PlayerDataStore getGlobalStore() {
+		return getStore(UUID.fromString("00000000-0000-0000-0000-000000000000"));
```

Comment:
simpler: `new UUID(0,0)`, maybe you should put the UUID of the global store into a constant